server/form-pages/apply/risk-and-need-factors/further-considerations/contingencyPlanPartners.test.ts

```diff
@@ -42,6 +48,23 @@ describe('ContingencyPlanPartners', () => {
         roleInPlan: 'You must specify a role in plan',
       })
     })
+
+    it('should return an error when there are no partner agencies added and saveAndContinue is truthy', () => {
+      const page = new ContingencyPlanPartners({ saveAndContinue: '1' } as Body)
+
+      expect(page.errors()).toEqual({
+        partnerAgencyDetails: 'You must add at least one partner agency',
+      })
+    })
+
+    it('shouldnt return an error if partnerAgencyDetails are added saveAndContinue is truthy', () => {
```

Comment:
"should not return an error if partnerAgencyDetails are added if(?) saveAndContinue is truthy"